homeassistant/components/microsoft_face/__init__.py

```diff
@@ -313,10 +337,12 @@ async def call_api(self, method, function, data=None, binary=False, params=None)
             )
             raise HomeAssistantError(answer["error"]["message"])
 
-        except aiohttp.ClientError:
-            _LOGGER.warning("Can't connect to microsoft face api")
+        except aiohttp.ClientError as err:
+            _LOGGER.error("Can't connect to microsoft face api with error: %s", err)
+            raise
 
         except asyncio.TimeoutError:
-            _LOGGER.warning("Timeout from microsoft face api %s", response.url)
+            _LOGGER.warning("Timeout from microsoft face api with '%s'", response.url)
+            raise
 
         raise HomeAssistantError("Network error on microsoft face api.")
```

Comment:
Here we're raising. Note that with the current change this code is now unreachable.

Reply:
Indeed. Fixed with new commit.